exe/etcd-env

```diff
@@ -5,14 +5,17 @@ require 'json'
 require 'net/http'
 
 def usage
-  puts "Usage: #{$PROGRAM_NAME} <etcd key> <command>"
+  puts "Usage: #{$PROGRAM_NAME} <etcd key> [ <etcd key> ... ] -- <command>"
 end
 
-def env_vars(key)
-  response = fetch "http://#{etcd_host}:#{etcd_port}/v2/keys#{key}"
-  (JSON.parse(response)['node']['nodes'] || [])
-    .map { |var| [var['key'].split('/').last, var['value']] }
-    .to_h
+def env_vars(keys)
+  keys.map! do |key|
```

Comment:
Better to use `map` than `map!` here - it's generally not a good idea to mutate function arguments. Here it doesn't affect the output, but let's do it anyway.

Or better still, you could just use `reduce` directly. No need for a separate step.